src/ftxui/component/dropdown.cpp

```diff
@@ -58,16 +58,22 @@ Component Dropdown(ConstStringListRef entries, int* selected) {
       });
     }
 
-    bool OnEvent(ftxui::Event event) override
-    {
-      if (event == Event::Return && show_)
-      {
-        children_[0]->OnEvent(event);
+    // Switch focus in between the checkbox and the radiobox when selecting it.
+    bool OnEvent(ftxui::Event event) override {
+      const bool show_old = show_;
+      const int selected_old = *selected_;
+      const bool handled = ComponentBase::OnEvent(event);
+
+      if (!show_old && show_) {
```

Comment:
I don't understand this line... why is this required?

Reply:
That's when the dropdown is activated, we put the focuse to the list. It avoid the user to press a key

Reply:
and what's the const bool show_old for?